project/plugins.sbt

```diff
@@ -10,7 +10,7 @@ addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "1.15.0")
 
 addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "1.2.0")
 
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.5.12")
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.5.19")
```

Comment:
im unsure of the repocussions of uplifting this without upgrading to play 2.5.19. i would revert until we do security upgrade